packages/components/statistic/__tests__/countdown.test.tsx

```diff
@@ -1,6 +1,11 @@
 import { mount } from '@vue/test-utils'
 import { describe, expect, it } from 'vitest'
+import dayjs from 'dayjs'
 import Countdown from '../src/countdown.vue'
+
+const wait = (ms = 100) =>
+  new Promise((resolve) => setTimeout(() => resolve(0), ms))
+
```

Comment:
There's a ready-made function in `packages/test-utils/sleep.ts`.

Reply:
adjusted.